docs/checkouts/lifecycle.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Checkout lifecycle
+sidebar_label: Lifecycle
+description: Understand a Checkout lifecycle
+keywords:
+  - checkouts
+  - features
+  - lifecycle
+---
+
+import Figure from "@site/src/components/Figures";
+import CheckoutStatuses from "@site/static/img/checkout_statuses.png";
+
+# Lifecycle
+
+## Overview
+
+A Checkout is not a Payment; it has its own data model, lifecycle, and statuses.
```

Comment:
What do you think about adding a bit more here after this sentence, like:
'It contains the information necessary to prepare for possibly multiple payment transactions.
During checkout completion a payment is created, processed and linked to the checkout.